### ISSUE TITLE
NullPointerException on Launch

### ISSUE THREAD

**Issue by Blue64** [konwboj/MagicClover#5]
I shall upload the crash log shortly, but SOMETHING is causing your mod specifically to crash hard during launch (will post specs too with Crash Log, it's for 1.7.10 By The Way)

The specific ref is
```
at clover.common.util.Registry.load(Registry.java:66)
At clover.common.MagicClover.postInit(MagicClover.java:57)
```
I'm going to go connect to Wi-Fi so I can upload my crash log to my github for you to fork and peruse at your discretion.

I should be able to inform you which mod is throwing the error by tomorrow too (it's 118 potential mods)

**Comment by Blue64** [comment]
crash log uploaded ***[here](https://github.com/Blue64/Misc/blob/master/Magic%20Clover%20Crash%20Log.txt)***.

**Comment by konwboj** [comment]
Thanks for the report. Fixed in 0.7. Going to release it tommorow.

**Comment by Blue64** [comment]
Cool, thanks. I was actually just thinking about this, I have not yet had a chance to get around to narrowing down the mod causing the problem, had one problem after another, still setting up the Config files (had to practically delete the entire Config folder 3 times in 1 month lol)
